packages/oid4vci-demo-frontend/src/configs/account.banqup.json

```diff
@@ -1,6 +1,6 @@
 {
   "general": {
-    "credentialName": "Account",
+    "credentialName": "Verified Account",
```

Comment:
Is is going to cause a problem with resource "credentials_success_right_pane_valid_id_title": 
"You have received your Verified {{name}} credential",
-> 
"You have received your Verified Verified Account credential",
One of both "Verifed" will have to go.

Same here:
"credentials_success_right_pane_registered_company_paragraph": "\n\n\nDe geverifieerde {{credentialName}} credential bewijst dat\nUnifiedPost heeft kunnen verifiëren dat je bedrijf een geregistreerd bedrijf is.\r\n\nJE kunt deze credential gebruiken om je UnifiedPost-account af te handelen.",

The Verified Verified credential proofs that
De geverifieerde Verifed Account credential bewijst dat...

(The only place EcosystemGeneralConfig.credentialName is used is in the language resources. EcosystemGeneralConfig.credentialName is actually the wrong place if once we can select from multiple credentials within one flow.)